bower.json

```diff
@@ -19,8 +19,9 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.6",
-    "paper-styles": "PolymerElements/paper-styles#^0.8.0"
+    "iron-input": "PolymerElements/iron-input#^0.8.0",
+    "paper-styles": "PolymerElements/paper-styles#^0.8.0",
+    "polymer": "Polymer/polymer#v0.8.0-rc.7"
```

Comment:
Polymer is the best dependency and deserves to go first (jk only sort of not jk).